Jenkinsfile

```diff
@@ -5,17 +5,40 @@ elifeLibrary {
         commit = elifeGitRevision()
     }
 
-    stage 'Build image', {
-        sh 'docker build -t elife/sciencebeam-utils .'
-    }
+    node('containers-jenkins-plugin') {
+        def candidateVersion
+        stage 'Build images', {
+            checkout scm
+            dockerComposeBuild(commit)
+            try {
+                candidateVersion = sh(script: "IMAGE_TAG=${commit} docker-compose run --rm ./print_version.sh", returnStdout: true).trim()
```

Comment:
you are using a `docker-compose.ci.yml` file but it's not being passed to the `docker-compose` `sh` commands. Works for now, but it may break if a service is added there

Reply:
good point, I guess the same is true for running the tests.

Reply:
And docker-compose down as well? (i.e. if the .ci version did contain additional services)